rvip/include/opengr/super4pcs.h

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Could you clarify why this file is copied here into this package?

It appears to be part of either PCL or Super4PCS?

Reply:
This file provides a streamlined abstraction to interact with the `opengr` library's API.

This file is temporary and will be removed in future iteration of the project.